components/core-jobs/src/main/java/org/eclipse/dirigible/components/jobs/synchronizer/JobSynchronizer.java

```diff
@@ -173,6 +186,19 @@ public void cleanup(Job artefact) {
      */
     @Override
     public boolean complete(TopologyWrapper<Artefact> wrapper, String flow) {
+        Job job = null;
+        if (wrapper.getArtefact() instanceof Job){
+            job = (Job) wrapper.getArtefact();
+            Job existingJob = jobService.findByName(job.getName());
+            List<JobEmail> emailDefinitions = jobEmailService.findAllByName(existingJob.getName());
+
+            jobLogService.jobLogged(existingJob.getName(), existingJob.getHandler(), existingJob.getMessage());
+            JobEmailProcessor.createAndSendJobEmail(job, existingJob, emailDefinitions);
```

Comment:
does not seem to be correct to use processor here
it should just cover the LM of the Job artefact